src/changelist/_format.py

```diff
@@ -3,21 +3,71 @@
 from collections import OrderedDict
 from collections.abc import Iterable
 from dataclasses import dataclass
-from typing import Union
+from datetime import datetime
 
 from github.NamedUser import NamedUser
 from github.PullRequest import PullRequest
 
 logger = logging.getLogger(__name__)
 
 
+@dataclass(frozen=True, slots=True)
+class ChangeItem:
+    content: str
+    reference_name: str
+    reference_url: str
+    labels: tuple[str]
+    timestamp: datetime
+
+    @classmethod
+    def prs_to_items(
+        cls,
+        pull_requests: set[PullRequest],
+        pr_summary_regex: re.Pattern,
+    ) -> "set[ChangeItem]":
+        items = set()
+        for pr in pull_requests:
+            matches = tuple(pr_summary_regex.finditer(pr.body))
+            if not matches:
+                logger.debug("falling back to title for %s", pr.html_url)
+                matches = [{"summary": pr.title, "label": None}]
```

Comment:
If `LABEL: ` is given in the PR title (e.g. the convention in the NumPy) and changelist falls back to it for the summary, the behavior is unchanged and it's included directly.